optuna/storages/base.py

```diff
@@ -98,8 +99,8 @@ def get_all_study_summaries(self):
     # Basic trial manipulation
 
     @abc.abstractmethod
-    def create_new_trial_id(self, study_id):
-        # type: (int) -> int
+    def create_new_trial_id(self, study_id, base_trial=None):
```

Comment:
`base_trial` reminds me [`BaseTrial class`](https://github.com/pfnet/optuna/blob/master/optuna/trial.py#L25) but it is actually an instance of `FixedTrial`. Do you have any idea about alternative names?

Reply:
I renamed it to `template_trial` (https://github.com/pfnet/optuna/pull/464/commits/7b405ae6c0292cdf5341a6768994687de3dbbf23).
Please let me know if you come up with a better name.